docs/source/daml/reference/contract-keys.rst

```diff
@@ -62,6 +62,21 @@ Like ``fetch``, ``fetchByKey`` needs to be authorized by at least one stakeholde
 
 This means that if it fails, it doesn't guarantee that a contract with that key doesn't exist, just that the submitting Party doesn't know about it, or there are issues with authorization.
 
+.. _doescontractexistforkey:
+
+canSeeContractWithKey
+=====================
+
+``boolean <- canSeeContractWithKey @ContractType contractKey``
+
+Use ``canSeeContractWithKey`` to check whether, in the current context, you can see an active contract for the current key. If the contract exists and the current context has the permission to see it, returns ``True``, otherwise returns ``False``.
```

Comment:
I don’t think we use `context` anywhere else. This should probably say something like “with the current authorization”.

Reply:
Done.